Testscripts/Linux/gpu-driver-install.sh

```diff
@@ -169,6 +178,13 @@ function InstallGRIDdrivers() {
 EOF
 	LogMsg "Updated nouveau.conf file with blacklist"
 
+	# Unload nouveau in RHEL-8 or it will block the GRID driver installation
+	# https://access.redhat.com/solutions/1155663
+	if [[ $DISTRO == "redhat_8" ]]; then
+		rmmod nouveau
+		LogMsg "Unload nouveau driver"
+	fi
+
```

Comment:
in function install_gpu_requirements, you already run this?

Reply:
Ah yes you're right! Removed this one. Thanks for reminding!